server/src/main/java/org/candlepin/model/Product.java

```diff
@@ -77,10 +77,75 @@
 @Entity
 @Table(name = "cp2_products")
 public class Product extends AbstractHibernateObject implements SharedEntity, Linkable, Cloneable {
-    public static final String UEBER_PRODUCT_POSTFIX = "_ueber_product";
+    /**
+     * Commonly used/recognized product attributes
+     */
+    public static final class Attributes {
+        /** Attribute to specify the architecture on which a given product can be installed/run */
+        public static final String ARCHITECTURE = "arch";
+
+        /** Attribute for specifying the type of branding to apply to a marketing product (SKU) */
+        public static final String BRANDING_TYPE = "brand_type";
+
+        /** Attribute for enabling content overrides */
+        public static final String CONTENT_OVERRIDE_ENABLED = "content_override_enabled";
+
+        /** Attribute for disabling content overrides */
+        public static final String CONTENT_OVERRIDE_DISABLED = "content_override_disabled";
+
+        /** Attribute specifying the number of cores that can be covered by an entitlement using the SKU */
+        public static final String CORES = "cores";
+
+        /** Attribute specifying whether or not derived pools created for a given product will be
+         *  host-limited */
+        public static final String HOST_LIMITED = "host_limited";
+
+        /** Attribute used to specify the instance multiplier for a pool */
+        public static final String INSTANCE_MULTIPLIER = "instance_multiplier";
+
```

Comment:
Perhaps add: Triggers instance based subscriptions. Multiplies size of the pool, but physical binds will consume entitlements in multiples of this value.